From e39b143aa70df56e7aec899c26d7f1944c7135b3 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 21 Apr 2023 15:51:56 +0100
Subject: [PATCH] dtoverlays: Change i2c-fan pwm-max/min overrides to u8

The driver is only reading a u8, so change the overlay to match.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/i2c-fan-overlay.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/i2c-fan-overlay.dts b/arch/arm/boot/dts/overlays/i2c-fan-overlay.dts
index ee25780a2810..f2f4a2aa797a 100644
--- a/arch/arm/boot/dts/overlays/i2c-fan-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-fan-overlay.dts
@@ -94,8 +94,8 @@ __overrides__ {
 		i2c6 = <&frag100>, "target?=0",
 		       <&frag100>, "target-path=i2c6";
 		addr =		<&emc2301>,"reg:0";
-		minpwm =	<&emc2301>,"emc2305,pwm-min;0";
-		maxpwm =	<&emc2301>,"emc2305,pwm-max;0";
+		minpwm =	<&emc2301>,"emc2305,pwm-min.0";
+		maxpwm =	<&emc2301>,"emc2305,pwm-max.0";
 		midtemp =	<&fanmid0>,"temperature:0";
 		midtemp_hyst =	<&fanmid0>,"hysteresis:0";
 		maxtemp =	<&fanmax0>,"temperature:0";
-- 
2.35.5

